### PR TITLE
DP-26574: Pass focal point information to illustrated headers when used as a background image

### PR DESCRIPTION
**Pull request by justafish**
This PR requires this change to Mayflower: https://github.com/massgov/mayflower/pull/1750

**Description:**
Adds a helper function which fetches the focal point information for an image and then passes it along to templates as a percentage that can be used in css `background-position`.

**Jira:** (Skip unless you are MA staff)
DP-26574

**To Test:**
- [ ] Visit https://mass.local/info-details/adaptive-paddling and resize the browser. Note that the photo of the canoeist slides off to the side
  Screenshot from 2023-03-08 11-59-51
- [ ] Switch to this PR and run `composer install` which swaps `docroot/libraries/mayflower-artifacts/twig/03-organisms/by-template/illustrated-header.twig` with the version in https://github.com/massgov/mayflower/pull/1750
- [ ] Clear cache
- [ ] Refresh https://mass.local/info-details/adaptive-paddling and see the canoeist now remains in view
Screenshot from 2023-03-08 11-47-57

Can also be seen here: https://pr1920-iihovbcowjpobx1ici2l46oc5wle8iox.tugboatqa.com/info-details/adaptive-paddling

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: changelogs/DP-24477.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fixed:
+  - description: Passes focal point information to illustrated header background images
+    issue: DP-24477
```
Comment (justafish) [inline changelogs/DP-24477.yml]: I'll fix this being the wrong issue once the mayflower PR is merged

Comment (clairesunstudio) [inline changelogs/DP-24477.yml]:
```suggestion
    issue: DP-26574
```

**Comment by justafish** [comment]
@clairesunstudio I've updated this PR to the latest mayflower develop branch

**Review by clairesunstudio — APPROVED** [review]

**Comment by clairesunstudio** [comment]
Ignoring the backstop failure for now.

**Pull request merged**